kie-api/src/main/java/org/kie/api/management/ReleaseIdInfo.java

```diff
@@ -0,0 +1,76 @@
+/*
+ * Copyright 2015 Red Hat, Inc. and/or its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * 
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+*/
+
+package org.kie.api.management;
+
+import javax.management.openmbean.CompositeData;
+
+import org.kie.api.builder.ReleaseId;
+
+public class ReleaseIdInfo implements ReleaseId {
```

Comment:
(speaking with Edson, this will be renamed simply to `GAV` - dropping "Coordinates" postfix)